deep/permissions.py

```diff
@@ -167,4 +167,9 @@ def has_permission(self, request, view):
             return True
         elif entry_id and Entry.get_for(request.user).filter(id=entry_id).exists():
             return True
-        return False
+        return True
```

Comment:
This will always return **True** even if the member is not part of the project.

If `project_id` `lead_id` `entry_id` is defined then we should make sure the user is part of the project. For that, change the logic above to something like this.
```python
        if project_id:
            return Project.get_for_member(request.user).filter(id=project_id).exists()
```